lambda-layer/src/otel-instrument

```diff
@@ -108,13 +108,27 @@ fi
 
 export LAMBDA_RESOURCE_ATTRIBUTES="cloud.region=$AWS_REGION,cloud.provider=aws,faas.name=$AWS_LAMBDA_FUNCTION_NAME,faas.version=$AWS_LAMBDA_FUNCTION_VERSION,faas.instance=$AWS_LAMBDA_LOG_STREAM_NAME,aws.log.group.names=$AWS_LAMBDA_LOG_GROUP_NAME";
 
+
+if [ -z "${OTEL_AWS_APPLICATION_SIGNALS_ENABLED}" ]; then
+    export OTEL_AWS_APPLICATION_SIGNALS_ENABLED="true";
+fi
+
 # - If Application Signals is enabled
 
 if [ "${OTEL_AWS_APPLICATION_SIGNALS_ENABLED}" = "true" ]; then
     export OTEL_PYTHON_DISTRO="aws_distro";
     export OTEL_PYTHON_CONFIGURATOR="aws_configurator";
-    export OTEL_METRICS_EXPORTER="none";
-    export OTEL_LOGS_EXPORTER="none";
+    if [ -z "${OTEL_METRICS_EXPORTER}" ]; then
+        export OTEL_METRICS_EXPORTER="none";
+    fi
+fi
+
+# - If Application Signals is disabled
+
+if [ "${OTEL_AWS_APPLICATION_SIGNALS_ENABLED}" = "false" ]; then
+    if [ -z "${OTEL_METRICS_EXPORTER}" ]; then
+        export OTEL_METRICS_EXPORTER="none";
```

Comment:
why do we disable OTel Metrics exporter when Application Signals is not enabled?